src/screens/SUE/SueReportScreen.tsx

```diff
@@ -42,16 +42,24 @@ type TRequiredFields = {
   };
 };
 
+const keyMapping = {
```

Comment:
please add a comment why we need this mapping and when it can be deprecated

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/61153063342c5a49bf0e0619a5c66640bd732f5c